### PR TITLE
chore(external docs): Add `remove_tags` transform metadata

### PR DESCRIPTION
**Pull request by binarylogic**
Signed-off-by: binarylogic <bjohnson@binarylogic.com>

### CODE REVIEW

File: docs/reference/components/transforms/remove_tags.cue [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  }
+
+  support: {
+      input_types: ["metric"]
+
+    platforms: {
+      "aarch64-unknown-linux-gnu": true
+      "aarch64-unknown-linux-musl": true
+      "x86_64-apple-darwin": true
+      "x86_64-pc-windows-msv": true
```
Comment (Hoverbear) [inline docs/reference/components/transforms/remove_tags.cue:30]:
```suggestion
      "x86_64-pc-windows-msvc": true
```

File: docs/reference/components/transforms/remove_tags.cue [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package metadata
+
+components: transforms: remove_tags: {
+  title: "#{component.title}"
```
Comment (Hoverbear) [inline docs/reference/components/transforms/remove_tags.cue:4]:
```suggestion
  title: "remove_tags"
```

**Comment by binarylogic** [comment]
@Hoverbear feel free to take this over and make the changes as noted in #4303 😄 .

**Comment by Hoverbear** [comment]
Ok! :)

**Review by Hoverbear — APPROVED** [review]

File: docs/reference/components/transforms/remove_tags.cue [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      "aarch64-unknown-linux-gnu": true
+      "aarch64-unknown-linux-musl": true
+      "x86_64-apple-darwin": true
+      "x86_64-pc-windows-msv": true
+      "x86_64-unknown-linux-gnu": true
+      "x86_64-unknown-linux-musl": true
+    }
+
+    requirements: []
+    warnings: []
```
Comment (Hoverbear) [inline docs/reference/components/transforms/remove_tags.cue:33]: Ah I need to add `notices` or it will break

**Pull request merged**